pkg/player/player_desktop.go

```diff
@@ -0,0 +1,40 @@
+//go:build (linux || darwin) && !android
```

Comment:
I feel that i just broke windows here since it wont be included when GOOS is windows. Need add windows in or condition as well.

```go
//go:build (linux || darwin || windows) && !android
```

Reply:
kinda annoying to list it all here

Reply:
Fixed in #16